subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -362,17 +373,51 @@ int aws_fota_mqtt_evt_handler(struct mqtt_client *const client,
 			return 1;
 		}
 
-		err = aws_jobs_subscribe_topic_notify_next(client,
-							   notify_next_topic);
-		if (err) {
-			LOG_ERR("Unable to subscribe to notify-next topic");
-			return err;
-		}
+		session_valid = (evt->param.connack.session_present_flag != 0);
 
-		err = aws_jobs_subscribe_topic_get(client, "$next", get_topic);
-		if (err) {
-			LOG_ERR("Unable to subscribe to jobs/$next/get");
-			return err;
+		if (!session_valid) {
```

Comment:
I'm a bit curious why you chose to implement this as a global variable while in `aws_iot` just used the event directly? Is it that this flag is only set  on CONACK?

Reply:
Hmm, not sure why I did that.  I think I thought I needed it in some other phase of the event handler but I guess not.  I'll make it local.

Reply:
Done.